.github/workflows/int_test.yml

```diff
@@ -297,33 +311,43 @@ jobs:
           az iot ops broker stats
           az iot ops broker stats --raw
           az iot ops asset query -g ${{ env.RESOURCE_GROUP }} --location westus -o table
-          az iot ops verify-host
+          # az iot ops verify-host
```

Comment:
What was the reason?

Reply:
probably leftover from commenting out broken stuff...